services/server/config/config.go

```diff
@@ -165,6 +167,14 @@ type ProxyPlugin struct {
 	Address string `toml:"address"`
 }
 
+// OpenTelemetryConfig provides open telemetry configuration
+type OpenTelemetryConfig struct {
+	ServiceNameKey     string  `toml:"service_name"`
+	ExporterName       string  `toml:"exporter_name"`
+	ExporterEndPoint   string  `toml:"exporter_endpoint"`
```

Comment:
- ServiceNameKey -> ServiceName. I'd like to make it consistent with the TOML key and ExporterName (which doesn't have `Key`)
- ExporterEndPoint -> ExporterEndpoint. I'd like to make it consistent with the TOML key.

Reply:
Fixed the names to be consistent with TOML